scripts/build.sh

```diff
@@ -0,0 +1,3 @@
+#/bin/bash
```

Comment:
Is this a legit shebang?
Or should it be `#!/bin/bash` (or even `#!/usr/bin/env bash`)...